### DISCUSSION TITLE
唉唉

### DISCUSSION THREAD

**Discussion by zzz20100624** [share121/inter-knot#5028] [其他]
已经更到最新版了，怎么还是要重新登录啊

**Comment by KELERSAMA** [comment]
更新公告里说了 恢复8小时重新登陆
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
